active/social-media.md

```diff
@@ -0,0 +1,80 @@
+# Social Media Working Group
+
+## Scope of responsibilities
+
+This working group will manage Django's official social media profiles on behalf of the Django Software Foundation.
+
+The goals of the group are:
+- Promote the use of Django among the World Wide Web development community.
+- Protect the framework's long-term viability by generating revenue through sponsor relationships and campaigns.
+- Assist the board in posting official news and updates from the DSF.
+- Maintain and monitor automation in connection with posting DSF news and updates to social media channels.
```

Comment:
🤔 This sounds more like a responsibility rather than goal.

Reply:
Tried to shift some things to the the 'Delegated responsibilities' section 👍